crypto/thread/task.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/configuration.h>
+#include <openssl/crypto.h>
+
+#if defined(OPENSSL_THREADS)
+
+# include "task.h"
+
+static void crypto_task_free(CRYPTO_TASK *t);
+
+int task_list_cmp(struct list *l, void *data)
```

Comment:
Add `ossl_` prefix to non-static functions in this file.

Reply:
Resolved here: https://github.com/openssl/openssl/pull/12255/commits/0c644027aa27cd1acf8f28d721b3e368891b9eb0